Automattic/wp-calypso
# GS on Personal: Add Calypso Feature Flag

Related to:
- https://github.com/Automattic/dotcom-forge/issues/9517
## Proposed Changes

* Created a new feature flag `global-styles/on-personal-plan`

## Why are these changes being made?


* We need this feature flag so that we can develop the feature without affecting production.

## Testing Instructions



* Checkout the code and run `yarn feature-search global-styles/on-personal-plan`
* The feature should be enabled on development, and set to false in the rest of WPCom envs (A4A and Jetpack Cloud are excluded)

![Screenshot 2024-10-30 at 12 27 51](https://github.com/user-attachments/assets/67dfba6f-baf1-435d-af41-5887b20de5ed)


## Pre-merge Checklist
